### PR TITLE
[INLONG-585][Doc] Add document for multiple sink of Doris

### LINKED ISSUE #585: [Feature][Doc] Add document for multiple sink of Doris
### Description

Add document for multiple sink of Doris

### Use case

multi sink doris

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kuansix**
### Prepare a Pull Request

Title: [INLONG-585][Doc] Add document for multiple sink of Doris

- Fixes #585 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/data_node/load_node/doris.md [hunk 1]
```diff
@@ -134,7 +215,54 @@ Flink SQL> insert into cdc_doris_sink select * from cdc_mysql_source /*+ OPTIONS
...
+    >  'password' = '000000',
+    >  'sink.enable-delete' = 'true',
+    >  'sink.multiple.enable' = 'true',
+    >  'sink.multiple.format' = 'canal-json',
+    >  'sink.multiple.database-pattern' = '${database}',
+    >  'sink.multiple.table-pattern' = 'doris_${table}'
+    > );
+[INFO] Execute statement succeed.
+
+-- 支持删除事件同步(sink.enable-delete='true'), 需要 Doris 表开启批量删除功能
```
Comment (yunqingmoswu) [inline docs/data_node/load_node/doris.md:261]: It is required to translate.

File: docs/data_node/load_node/doris.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ sidebar_position: 16
 import {siteVariables} from '../../version';
 
 ## Overview
-
-The `Doris Load` node supports writing data to the Doris database. This document describes how to set up a
-Doris Load node to run SQL queries against the Doris database.
+ - `Doris Load` node supports writing data to the Doris database.
+ - Two modes are supported for sink to Doris: 
+Single-sink for specify fixed database name and table name to sink. 
+Multi-sink for custom database name and table name according to src format, which suitable for scenarios such as multi-table writing or whole database synchronization.
```
Comment (yunqingmoswu) [inline docs/data_node/load_node/doris.md:12]: synchronization -> writing

File: docs/data_node/load_node/doris.md [hunk 3]
```diff
@@ -59,18 +61,65 @@ mysql> select * from cdc_mysql_source;
 |  2 | lisi     |  0 |
 |  3 | wangwu   |  0 |
 +----+----------+----+
-3 rows in set (0.07 sec)     
+3 rows in set (0.07 sec)
+```
+##### 2.create tables `user_db.user_id_name`、`user_db.user_id_name` in the MySql database for multi-sink extract. the command is as follows:
```
Comment (yunqingmoswu) [inline docs/data_node/load_node/doris.md:66]: MySql -> MySQL

Comment (yunqingmoswu) [inline docs/data_node/load_node/doris.md:66]: create tables -> Create tables

File: docs/data_node/load_node/doris.md [hunk 4]
```diff
@@ -33,11 +35,11 @@ such as Maven or SBT is provided below.
 </code></pre>
 
 ## Prepare
-### Create a MySql Extract table
-First create a table `cdc_mysql_source` in the MySql database, the command is as follows:
+### Create MySql Extract table
+##### 1.create a table `cdc.cdc_mysql_source` in the MySql database for single-sink extract. the command is as follows:
```
Comment (yunqingmoswu) [inline docs/data_node/load_node/doris.md:39]: The first letter should be upper for the first word.

**Review by yunqingmoswu — COMMENTED** [review]
一、英文文档
1.首字母大写
2.尽量不使用数字标题

二.中文文档：
1.中英文直接需要空格隔开，英文首字母大写。

File: docs/data_node/load_node/doris.md (hunk 2 above)
Comment (kuansix) [reply docs/data_node/load_node/doris.md:12]: synchronization is OK?

File: docs/data_node/load_node/doris.md (hunk 1 above)
Comment (kuansix) [reply docs/data_node/load_node/doris.md:261]: done

File: docs/data_node/load_node/doris.md (hunk 3 above)
Comment (kuansix) [reply docs/data_node/load_node/doris.md:66]: done

File: docs/data_node/load_node/doris.md (hunk 2 above)
Comment (yunqingmoswu) [reply docs/data_node/load_node/doris.md:12]: 👌

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md [hunk 5]
```diff
@@ -82,11 +131,43 @@ mysql> CREATE TABLE `cdc_doris_sink` (
...
+       "replication_allocation" = "tag.location.default: 1"
+       );
+Query OK, 0 rows affected (0.06 sec)
+```
 
 ## 如何创建 Doris Load 节点
 
 ### SQL API 用法
-
+- 单表写入：doris单表写入
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md:170]: 中英文用空格隔开，首字母大写

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md [hunk 6]
```diff
@@ -132,7 +212,55 @@ Flink SQL> insert into cdc_doris_sink select * from cdc_mysql_source /*+ OPTIONS
 [INFO] Submitting SQL update statement to the cluster...
 [INFO] SQL update statement has been successfully submitted to the cluster:
 Job ID: 5f89691571d7b3f3ca446589e3d0c3d3
+```
+
+- 多表写入：doris多表写入
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md:217]: 中英文用空格隔开，首字母大写

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md [hunk 7]
```diff
@@ -168,6 +296,13 @@ TODO: 将在未来支持此功能。
...
 | sink.enable-delete                | 可选   | true              | boolean  | 是否启用删除。此选项需要 Doris 表开启批量删除功能(0.15+版本默认开启)，只支持 Uniq 模型。                                                                                                                                                                                                                                                 |
+| sink.enable-delete                | 可选   | true              | boolean  | 是否启用删除。此选项需要 Doris 表开启批量删除功能(0.15+版本默认开启)，只支持 Uniq 模型。                                                                                                                                                                                                                                                 |
+| sink.multiple.enable              | 可选   | false             | boolean  | 是否支持Doris多表写入。`sink.multiple.enable`为`true`时，需要`sink.multiple.format`、`sink.multiple.database-pattern`、`sink.multiple.table-pattern` 分别设置正确的值。        |
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md:300]: 中英文用空格隔开

**Review by yunqingmoswu — COMMENTED** [review]
It is recommend to add describe for the whole migiration.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md (hunk 5 above)
Comment (kuansix) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md (hunk 6 above)
Comment (kuansix) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md (hunk 7 above)
Comment (kuansix) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/doris.md]: done

**Comment by kuansix** [comment]
> It is recommend to add describe for the whole migiration.

For Doris sink, whole migration is same with multi-sink. Maybe it is necessary to describe in the source doc?

**Review by yunqingmoswu — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**